llvm/test/CodeGen/X86/combine-rotates.ll

```diff
@@ -477,3 +451,6 @@ declare i5 @llvm.fshl.i5(i5, i5, i5)
 
 declare <4 x i32> @llvm.fshl.v4i32(<4 x i32>, <4 x i32>, <4 x i32>)
 declare <4 x i32> @llvm.fshr.v4i32(<4 x i32>, <4 x i32>, <4 x i32>)
+;; NOTE: These prefixes are unused and the list is autogenerated. Do not add tests below this line:
+; SSE2-LIS: {{.*}}
+; SSE2-LV: {{.*}}
```

Comment:
Remove these prefixes from the RUN lines?

Reply:
Done.